plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -228,7 +229,144 @@ function importSubmissions($importXml, $deployment) {
 	 * @copydoc ImportExportPlugin::executeCLI
 	 */
 	function executeCLI($scriptName, &$args) {
-		fatalError('Not implemented.');
+		$opts = $this->parseOpts($args, ['no-embed', 'use-file-urls']);
+		$command = array_shift($args);
+		$xmlFile = array_shift($args);
+		$pressPath = array_shift($args);
+
+		AppLocale::requireComponents(LOCALE_COMPONENT_APP_MANAGER, LOCALE_COMPONENT_PKP_MANAGER, LOCALE_COMPONENT_PKP_SUBMISSION);
+		$pressDao = DAORegistry::getDAO('PressDAO');
+		$userDao = DAORegistry::getDAO('UserDAO');
+		$press = $pressDao->getByPath($pressPath);
+
+		if (!$press) {
+			if ($pressPath != '') {
+				echo __('plugins.importexport.common.cliError') . "\n";
+				echo __('plugins.importexport.common.error.unknownPress', array('pressPath' => $pressPath)) . "\n\n";
+			}
+			$this->usage($scriptName);
+			return;
+		}
+
+		if ($xmlFile && $this->isRelativePath($xmlFile)) {
+			$xmlFile = PWD . '/' . $xmlFile;
+		}
+
+		switch ($command) {
+			case 'import':
+				$userName = array_shift($args);
+				$user = $userDao->getByUsername($userName);
+
+				if (!$user) {
+					if ($userName != '') {
+						echo __('plugins.importexport.common.cliError') . "\n";
+						echo __('plugins.importexport.native.error.unknownUser', array('userName' => $userName)) . "\n\n";
+					}
+					$this->usage($scriptName);
+					return;
+				}
+
+				if (!file_exists($xmlFile)) {
+					echo __('plugins.importexport.common.cliError') . "\n";
+					echo __('plugins.importexport.common.export.error.inputFileNotReadable', array('param' => $xmlFile)) . "\n\n";
+					$this->usage($scriptName);
+					return;
+				}
+
+				$filter = 'native-xml=>issue';
+				// is this articles import:
+				$xmlString = file_get_contents($xmlFile);
+				$document = new DOMDocument();
+				$document->loadXml($xmlString);
+				if (in_array($document->documentElement->tagName, array('article', 'articles'))) {
+					$filter = 'native-xml=>article';
+				}
+				$deployment = new NativeImportExportDeployment($press, $user);
+				$deployment->setImportPath(dirname($xmlFile));
+				$content = $this->importSubmissions($xmlString, $filter, $deployment);
+				$validationErrors = array_filter(libxml_get_errors(), function($a) {
+					return $a->level == LIBXML_ERR_ERROR || $a->level == LIBXML_ERR_FATAL;
+				});
+
+				// Are there any import warnings? Display them.
+				$errorTypes = array(
+					ASSOC_TYPE_ISSUE => 'issue.issue',
```

Comment:
Issues don't exist in OMP; this can be removed.

Reply:
done